localisation/00_zuwc_united_localisation.csv

```diff
@@ -447,9 +447,9 @@ EVTTITLE_WCTHW_1015;The Splintering of the Elves;L'
-EVTDESC_WCCLN_111;Unfortunately, your recent marriage proposal is impossible for your race, as it contradicts your values and/or physiology, so you have abandoned this effort.;Malheureusement, votre r�cente proposition de mariage est impossible pour votre race, car cela contredit vos valeurs et / ou votre physiologie, vous avez donc abandonn� ce projet.;Ungl�cklicherweise ist Eure j�ngste Ehe ein Ding der Unm�glichkeit f�r Eure Rasse, sie widerspricht all unseren Wertevorstellungen und unserer Physiologie, daher kann ich das nicht zulassen.;;;;;;;;;;;x
-EVTDESC_WCCLN_112;Unfortunately for you, your race or place in the court does not allow you to love someone or to get married to whom you wish, so you abandon this effort.;Malheureusement pour vous, votre race ou votre place � la cour ne vous autorisent pas � aimer quelqu'un ou � vous marier librement, vous avez donc abandonn� cet effort.;Ungl�cklicherweise erlaubt Eure Rasse oder Euer Platz an diesem Hof es Euch nicht, zu lieben oder zu heiraten, wen Ihr w�nscht, daher m�sst Ihr diese Idee aufgegeben.;;;;;;;;;;;x
-EVTDESC_WCCLN_113;Unfortunately, the marriage between [target_husband.GetDynName] and [target_wife.GetDynName] is impossible, so you have both abandoned the effort.;Malheureusement, le mariage entre [target_husband.GetDynName] et [target_wife.GetDynName] est impossible. Vous avez donc tous deux abandonn�s le projet.;Ungl�cklicherweise ist die Hochzeit zwischen [target_husband.GetDynName] und [target_wife.GetDynName] nicht m�glich, also habt ihr beiden diese Idee aufgegeben.;;;;;;;;;;;x
+EVTDESC_WCCLN_111;Unfortunately for you, your recent marriage proposal is impossible for your race, as it contradicts your moral values and/or physiology, so you abandon this effort.;Malheureusement, votre r�cente proposition de mariage est impossible pour votre race, car cela contredit vos valeurs et / ou votre physiologie, vous avez donc abandonn� ce projet.;Ungl�cklicherweise ist Eure j�ngste Ehe ein Ding der Unm�glichkeit f�r Eure Rasse, sie widerspricht all unseren Wertevorstellungen und unserer Physiologie, daher kann ich das nicht zulassen.;;;;;;;;;;;x
+EVTDESC_WCCLN_112;Unfortunately for you, your race or moral values does not allow you to love or marry someone you wish, so you abandon this effort.;Malheureusement pour vous, votre race ou votre place � la cour ne vous autorisent pas � aimer quelqu'un ou � vous marier librement, vous avez donc abandonn� cet effort.;Ungl�cklicherweise erlaubt Eure Rasse oder Euer Platz an diesem Hof es Euch nicht, zu lieben oder zu heiraten, wen Ihr w�nscht, daher m�sst Ihr diese Idee aufgegeben.;;;;;;;;;;;x
```

Comment:
I'd delete the "for you" part here, sounds a little odd.

Reply:
Changed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1281/commits/eb15e9c61d23c1f4f9b2150e934258c58ca859e4.